examples/rust/Cargo.toml

```diff
@@ -44,19 +18,28 @@ version = "1.0.160"
 features = ["derive"]
 optional = false
 
-[dependencies.siwe]
-version = "0.5.0"
-features = ["serde"]
+[dependencies.tonic]
+version = "0.9.1"
+features = ["tls"]
 optional = false
 
-[dependencies.tokio]
-version = "1.27"
-features = ["macros", "rt-multi-thread"]
+[dev-dependencies]
```

Comment:
Dev-dependencies are not used when compiling a package for building, but are used for compiling tests, examples, and benchmarks.

These changes were made since the dependencies were incorrect in the exist toml, only those now contained under dependencies are required.